src/Mod/Draft/DraftEdit.py

```diff
@@ -244,7 +245,7 @@ def keyPressed(self, event_callback):
             if key == 111: # "o"
                 self.finish(closed=True)
             if key == 105: # "i"
-                if Draft.getType(self.obj) == "Circle": self.arcInvert()
+                if Draft.getType(self.obj) == "Circle": self.arcInvert(self.obj)
```

Comment:
I suggest complying to PEP8, so avoid control structures in the same line. It's better to break them to two lines. More readable this way.
```py
if Draft.getType(self.obj) == "Circle":
    self.arcInvert(self.obj)
```